src/command/eventpoll/eventpoll_start.ts

```diff
@@ -1,16 +1,89 @@
 import type {
   ApplicationCommandOption,
   Bot,
+  Embed,
   Interaction,
 } from "../../../deps.ts";
-import type { CommandHandler, SubCommand } from "../../../shared.ts";
+import type { SubCommand, SubCommandArgument } from "../../../shared.ts";
 import type { Loggable } from "../../logger/mod.ts";
 
 import {
   ApplicationCommandOptionTypes,
   InteractionResponseTypes,
 } from "../../../deps.ts";
 
+import { AbstractSubCommand } from "../../../shared.ts";
+
+const emojis: string[] = [
+  "🇦", // :regional_indicator_a
+  "🇧", // :regional_indicator_b
+  "🇨", // :regional_indicator_c
+  "🇩", // :regional_indicator_d
+  "🇪", // :regional_indicator_e
```

Comment:
These characters are defined in Unicode.
https://en.wikipedia.org/wiki/Regional_indicator_symbol#Unicode_block